dev-python/nbconvert/nbconvert-9999.ebuild

```diff
@@ -22,9 +22,10 @@ fi
 
 LICENSE="BSD"
 SLOT="0"
-IUSE="test"
+IUSE="doc test"
 
 RDEPEND="
+	>=app-text/pandoc-1.12.1
```

Comment:
Alright, I pulled the commit that added this and added one that notes pandoc as an optional dependency.